### ISSUE TITLE
New Rule: Enforce Line Ending Style

### ISSUE THREAD

**Issue by Snugug** [sasstools/sass-lint#408]
We should provide a way to enforce the kind of line endings that should be used. An extension of of indentation I believe we talked about.

Name: `newline-style`
Options: `lf`, `cr`, `crlf`

**Comment by DanPurdy** [comment]
This would be good, would work well in conjunction with the indentation rule. We're seeing a lot of issues related to different line endings and indentation at the moment.

**Comment by Snugug** [comment]
Agreed. I think I'll jump on this so, ya know, I CAN CONTRIBUTE SOMETHING FOR A CHANGE!

:tongue:

**Comment by bgriffith** [comment]
Sounds good :+1:
